### PR TITLE
added code to count occurrence of a character

### PR DESCRIPTION
**Pull request by ShaluMishra13**

### CODE REVIEW

**Comment by Pradyuman7** [comment]
Thanks for contributing, but the code file looks empty, is that expected?

**Comment by ShaluMishra13** [comment]
i have wrote the code with explaination in the file.

File: ShaluMishra13/contributing.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ShaluMishra13) [inline ShaluMishra13/contributing.md:1]: // Java program to count frequencies of
// characters in string using Hashmap
import java.io.*;
import java.util.*;
class OccurenceOfCharInString {
	static void characterCount(String inputString)
	{
		// Creating a HashMap containing char
		// as a key and occurrences as a value
		HashMap<Character, Integer> charCountMap
			= new HashMap<Character, Integer>();

		// Converting given string to char array

		char[] strArray = inputString.toCharArray();

		// checking each char of strArray
		for (char c : strArray) {
			if (charCountMap.containsKey(c)) {

				// If char is present in charCountMap,
				// incrementing it's count by 1
				charCountMap.put(c, charCountMap.get(c) + 1);
			}
			else {

				// If char is not present in charCountMap,
				// putting this char to charCountMap with 1 as it's value
				charCountMap.put(c, 1);
			}
		}

		// Printing the charCountMap
		for (Map.Entry entry : charCountMap.entrySet()) {
			System.out.println(entry.getKey() + " " + entry.getValue());
		}
	}

	// Driver Code
	public static void main(String[] args)
	{
		String str = "Ajit";
		characterCount(str);
	}
}

**Comment by ShaluMishra13** [comment]
sorry i dont know how is it empty may be mistakenly I did it .  If possible im just adding it.

**Pull request closed without merging**